### PR TITLE
Refactor learning rate config, support 10 lr strategies

### PR DESCRIPTION
**Pull request by LutaoChu**

### CODE REVIEW

File: paddleseg/cvlibs/config.py [hunk 1]
```diff
@@ -136,8 +140,28 @@ def iters(self) -> int:
...
+                'No `lr_scheduler` specified in the configuration file.')
+        params = self.dic.get('lr_scheduler')
+
+        lr_type = params.pop('type')
+        if lr_type == 'PolynomialDecay':
+            params.setdefault('decay_steps', self.iters)
+            params.setdefault('end_lr', 0)
+            params.setdefault('power', 0.9)
+
+        return eval('paddle.optimizer.lr.' + lr_type + '(**params)')
```
Comment (nepeplwu) [inline paddleseg/cvlibs/config.py]: 建议改为
```python
return getattr(paddle.optimizer.lr, lr_type)(**params)
```

File: paddleseg/cvlibs/config.py [hunk 2]
```diff
@@ -136,8 +140,28 @@ def iters(self) -> int:
...
+
     @property
     def learning_rate(self) -> paddle.optimizer.lr.LRScheduler:
+        warnings.warn(
```
Comment (nepeplwu) [inline paddleseg/cvlibs/config.py]: 建议使用`logger.warning`来报错

File: configs/README.md [hunk 3]
```diff
@@ -28,14 +28,21 @@
...
-### learning_rate
+### lr_scheduler
 > 学习率
 >  * 参数
+>     * type : 学习率类型，支持10种策略，分别是'PolynomialDecay', 'PiecewiseDecay', 'StepDecay', 'CosineAnnealingDecay', 'ExponentialDecay', 'InverseTimeDecay', 'LinearWarmup', 'MultiStepDecay', 'NaturalExpDecay', 'NoamDecay'.
+>     * **others** : 请参考[Paddle官方LRScheduler文档](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/optimizer/lr/LRScheduler_cn.html)
+
+----
+### learning_rate
+> 学习率（不推荐使用该配置，将来会被废弃，建议使用`lr_scheduler`代替）
```
Comment (nepeplwu) [inline configs/README.md]: 括号内的注释建议移到标题上

File: paddleseg/cvlibs/config.py (hunk 1 above)
Comment (LutaoChu) [reply paddleseg/cvlibs/config.py]: done

File: paddleseg/cvlibs/config.py (hunk 2 above)
Comment (LutaoChu) [reply paddleseg/cvlibs/config.py]: done

File: configs/README.md (hunk 3 above)
Comment (LutaoChu) [reply configs/README.md]: done

File: paddleseg/cvlibs/config.py [hunk 4]
```diff
@@ -15,11 +15,13 @@
 import codecs
 import os
 from typing import Any
+import warnings
```
Comment (nepeplwu) [inline paddleseg/cvlibs/config.py]: 冗余import

Comment (LutaoChu) [reply paddleseg/cvlibs/config.py]: done
